Add Web Performance Budget (#1669)
sindresorhus/awesome: readme.md

- [Blazor](https://github.com/AdrienTorris/awesome-blazor#readme) - .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
- [Web Performance Metrics](https://github.com/csabapalfi/awesome-web-performance-metrics#readme) - Metrics to help understand page speed and user experience.
- [Tailwind CSS](https://github.com/aniftyco/awesome-tailwindcss#readme) - Utility-first CSS framework for rapid UI development.

> - [Web Performance Budget](https://github.com/pajaydev/awesome-performance-budget#readme) - Curated list of why, how and tools to implement Web Performance Budget.

Reviewer: > Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.

- [Web Performance Metrics](https://github.com/csabapalfi/awesome-web-performance-metrics#readme) - Metrics to help understand page speed and user experience.
- [Tailwind CSS](https://github.com/aniftyco/awesome-tailwindcss#readme) - Utility-first CSS framework for rapid UI development.
- [Seed](https://github.com/seed-rs/awesome-seed-rs#readme) - Rust framework for creating web apps running in WebAssembly.

> - [Web Performance Budget](https://github.com/pajaydev/awesome-performance-budget#readme) - Techniques to set and maintain pre-defined performance metrics for our page.

Reviewer: "our page". That's a weird way to put it.

```suggestion
- [Web Performance Budget](https://github.com/pajaydev/awesome-performance-budget#readme) - Techniques to ensure certain performance metrics for a website.
```

?

Author: Yup makes sense 👍
